MapboxNavigation/NavigationViewController.swift

```diff
@@ -75,12 +75,19 @@ public protocol NavigationViewControllerDelegate {
      
      If this method is unimplemented, the navigation map view draws the route line using an `MGLLineStyleLayer`.
      */
-    @objc optional func navigationMapView(_ mapView: NavigationMapView, routeStyleLayerWithIdentifier identifier: String, source: MGLSource) -> MGLStyleLayer?
+    @objc optional func navigationMapView(_ mapView: NavigationMapView, routeWaypointCircleStyleLayerWithIdentifier identifier: String, source: MGLSource) -> MGLStyleLayer?
+    
+    /**
+     Returns an `MGLStyleLayer` that determines the appearance of the route line.
+     
+     If this method is unimplemented, the navigation map view draws the route line using an `MGLLineStyleLayer`.
+     */
+    @objc optional func navigationMapView(_ mapView: NavigationMapView, routeWaypointSymbolStyleLayerWithIdentifier identifier: String, source: MGLSource) -> MGLStyleLayer?
     
```

Comment:
Then remove runtime styling in favor of annotations, because that’s the usual way people add markers to the map. The main reason to expose this customization hook is to allow the developer to make the destination marker match the annotation on the main map in their application (the map that isn’t NavigationMapView).